engines/scumm/actor.cpp

```diff
@@ -259,13 +259,125 @@ void Actor_v0::initActor(int mode) {
 		_limb_flipped[i] = false;
 	}
 
+	walkBoxQueueReset();
+
 	if (_vm->_game.features & GF_DEMO) {
 		_sound[0] = v0ActorDemoTalk[_number];
 	} else {
 		_sound[0] = v0ActorTalk[_number];
 	}
 }
 
+void Actor_v0::walkBoxQueueReset() {
+	_walkboxHistory.clear();
+	_walkboxQueueIndex = 0;
+
+	for (uint i = 0; i < ARRAYSIZE( _walkboxQueue ); ++i) {
```

Comment:
Nope, we are not putting spaces after and before the parens. Should be '(_walkboxQueue)', and further, and with '( box )' laterm etc.